src/routes/(app)/admin[[hash=admin_hash]]/arm/user/[id]/+page.server.ts

```diff
@@ -48,36 +42,27 @@ export const actions = {
 		if (user.roleId !== SUPER_ADMIN_ROLE_ID && !parsed.recoveryEmail && !parsed.recoveryNpub) {
 			throw error(400, 'You must provide a recovery email or npub');
 		}
-		if (user.roleId === SUPER_ADMIN_ROLE_ID) {
-			await collections.users.updateOne(
-				{ _id: user._id },
-				{
-					$set: {
-						...(parsedAdmin.alias && { alias: parsedAdmin.alias })
-					}
-				}
-			);
+		if (user.roleId === SUPER_ADMIN_ROLE_ID && locals.user?.roleId !== SUPER_ADMIN_ROLE_ID) {
+			throw error(400, 'You are not allowed to edit admin information');
```

Comment:
Small regression - before other admins could edit the alias of the super admin. But it's ok imo cc @Tirodem , up to you

Reply:
If the regression forbid for non SU admin to edit SU admin login, it's not a regression, it's a good feature 👌

Reply:
just the alias that showed up in the UI

Reply:
oh, damn, didn't understand that
/arm is a sensitive page so it's not shared to every profile (usually super-admin and another admin+) but it would be great to hide superadmin login for non-super-admin users with access to /arm
i'll keep this to create an improvement ticket